src/enum.cr

```diff
@@ -390,7 +379,7 @@ struct Enum
     {% if @type.annotation(Flags) %}
       return if value == 0
       {% for member in @type.constants %}
-        {% if member.stringify != "All" %}
+        {% if member.stringify != "All" && member.stringify != "None" %}
```

Comment:
`LLVM::Attribute#each_kind` does a similar thing and needs the same treatment

Reply:
Yeah, I suppose we can refactor `#each_kind`. That should be a separate change though.

Same for the method signature, it's unrelated to this change.

Reply:
`#each_kind`: https://github.com/crystal-lang/crystal/pull/13234 (this needs to be merged first)